lib/checks/checks.go

```diff
@@ -579,12 +569,12 @@ func (r *checker) checkBandwidth(ctx context.Context) error {
 
 	for addr, result := range resp {
 		ip, _ := utils.SplitHostPort(addr, "")
-		server, err := findServer(r.servers, ip)
+		server, err := findServer(servers, ip)
 		if err != nil {
 			return trace.Wrap(err)
 		}
 
-		requirements := r.requirements[server.Server.Role]
+		requirements := r.Reqs[server.Server.Role]
```

Comment:
nit: I'd simply export the old attribute: **R**equirements as abbreviation does not make it any clearer